src/common/obj_bencher.cc

```diff
@@ -376,7 +376,17 @@ int ObjBencher::fetch_bench_metadata(const std::string& metadata_file,
   } else {
     *op_size = *object_size;
   }
-  unsigned ops_per_object = *object_size / *op_size;
+
+  unsigned ops_per_object;
+
+  // in case *op_size came out wrong
+  if((*op_size == 0) || (*object_size < *op_size)) {
+    ops_per_object = 1;
+  }
+  else {
```

Comment:
please move `else` up to end of previous line to be consistent with the rest of the source tree.